core/shared/src/main/scala/org/http4s/MediaType.scala

```diff
@@ -263,14 +263,16 @@ object MediaType extends MimeDB {
 
   // Curiously text/event-stream isn't included in MimeDB
   lazy val `text/event-stream` = new MediaType("text", "event-stream")
+  // Not in MimeDB but recommended here https://graphql.org/learn/serving-over-http/#post-request
+  lazy val `application/graphql` = new MediaType("application", "graphql", true)
```

Comment:
I'd be happier with a named parameter here - otherwise looks good:

```suggestion
  lazy val `application/graphql` = new MediaType("application", "graphql", compressible = true)
```